src/libs/actions/Card.ts

```diff
@@ -0,0 +1,184 @@
+import Onyx from 'react-native-onyx';
+import * as API from '@libs/API';
+import * as Localize from '@libs/Localize';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type {Response} from '@src/types/onyx';
+
+function reportVirtualExpensifyCardFraud(cardID: number) {
+    type ReportVirtualExpensifyCardFraudParams = {
+        cardID: number;
+    };
+
+    const parameters: ReportVirtualExpensifyCardFraudParams = {
+        cardID,
+    };
+
+    API.write('ReportVirtualExpensifyCardFraud', parameters, {
+        optimisticData: [
+            {
+                onyxMethod: Onyx.METHOD.MERGE,
+                key: ONYXKEYS.FORMS.REPORT_VIRTUAL_CARD_FRAUD,
+                value: {
+                    isLoading: true,
+                },
+            },
+        ],
+        successData: [
+            {
+                onyxMethod: Onyx.METHOD.MERGE,
+                key: ONYXKEYS.FORMS.REPORT_VIRTUAL_CARD_FRAUD,
+                value: {
+                    isLoading: false,
+                },
+            },
+        ],
+        failureData: [
+            {
+                onyxMethod: Onyx.METHOD.MERGE,
+                key: ONYXKEYS.FORMS.REPORT_VIRTUAL_CARD_FRAUD,
+                value: {
+                    isLoading: false,
+                },
+            },
+        ],
+    });
+}
+
+/**
+ * Call the API to deactivate the card and request a new one
+ * @param cardId - id of the card that is going to be replaced
+ * @param reason - reason for replacement ('damaged' | 'stolen')
+ */
+function requestReplacementExpensifyCard(cardId: number, reason: string) {
```

Comment:
```suggestion
function requestReplacementExpensifyCard(cardId: number, reason: 'damaged' | 'stolen') {
```

As described in JSDoc `reason` can be `'damaged' | 'stolen'`. You can also find the available options [here](https://github.com/Expensify/App/blob/main/src/pages/settings/Wallet/ReportCardLostPage.js#L27-L36).